projekt_do_pracy_na_labach/fiszki_backend/Dockerfile

```diff
@@ -1,2 +1,28 @@
+#Bartosz Habas
+
 FROM python:3.10
 
+WORKDIR /app
+
+COPY . /app
+RUN  pip install -r requirements.txt
+EXPOSE 8000 
+
+CMD ["python", "main.py"]
+
+
+
+#docker build --tag fiszki_backend .
+
+#docker run fiszki_backend 
+
+#docker run -p 8000 fiszki_backend 
+
+#docker network create web_example
+
+#docker run --network=web_example fiszki_backend
+
+#docker run --network=web_example mongo
```

Comment:
Bardzo dobrze, ostatnia rzecz jaka została, to ogarnąć zmienną środowiskową DATABASE_URL tak jak wspomniałem w mailu / jest w instrukcji zaktualizowanej na masterze.